Libs/MRML/Widgets/qMRMLSliceControllerWidget.cxx

```diff
@@ -781,19 +782,59 @@ void qMRMLSliceControllerWidgetPrivate::updateWidgetFromMRMLSliceNode()
   // Update abbreviated slice view name
   this->ViewLabel->setText(this->MRMLSliceNode->GetLayoutLabel());
 
+  vtkNew<vtkStringArray> orientationNames;
+  this->MRMLSliceNode->GetSliceOrientationPresetNames(orientationNames.GetPointer());
+
+  wasBlocked = this->SliceOrientationSelector->blockSignals(true);
+  int count = this->SliceOrientationSelector->count();
+  if(this->SliceOrientationSelector->itemText(count - 1).compare("Reformat") &&
```

Comment:
`this->SliceOrientationSelector->itemText(count - 1).compare("Reformat")` -> `this->SliceOrientationSelector->itemText(count - 1) != "Reformat"`